index.bs

```diff
@@ -2196,7 +2196,7 @@ when invoked, must run these steps:
 
 1. If |version| is 0 (zero), [=throw=] a [=TypeError=].
 
-1. Let |origin| be the [=/origin=] of the global scope used to access **this**.
+1. Let |origin| be the [=/origin=] of the global scope used to access [=/this=].
```

Comment:
nit (but unrelated to this PR) this would probably be more precise as the origin of the relevant settings object for this (actually not sure which global/settings object this refers to...)

Reply:
Thanks. I'll tackle that separately.